### ISSUE TITLE
How can I load an obj file with .mtl which describe like this?

### ISSUE THREAD

**Issue by BreadLee** [andresoviedo/android-3D-model-viewer#62]
hello! thanks for your sharing! 
I want to load some .obj file , but all of their material(.mtl) cannot load successfully.
I have tried many ways ,but still fail to load their pic material or just can load the first part of the mtl.
And then I found the .mtl examples in your sharing are different from mine, so I think maybe it is the major problem, but I still can't solve it.
How can I modify my .mtl file or .obj file ,or how can I modify code based on your sharing?
Here is one of my .mtl file:

newmtl patty
	Ns 0
	d 1
	illum 2
	Kd 0.8 0.8 0.8
	Ks 0.0 0.0 0.0
	Ka 0.2 0.2 0.2
	map_Kd Patty_Wagon.png

newmtl pat
	Ns 0
	d 1
	illum 2
	Kd 0.8 0.8 0.8
	Ks 0.0 0.0 0.0
	Ka 0.2 0.2 0.2
	map_Kd Char_Patrick.png

newmtl sb
	Ns 0
	d 1
	illum 2
	Kd 0.8 0.8 0.8
	Ks 0.0 0.0 0.0
	Ka 0.2 0.2 0.2
	map_Kd Spongebob_SquarePants.png

Hope you can help me to solve my problem,thanks a lot! Have a nice day!

**Comment by andresoviedo** [comment]
Hi. If possible, please attach obj, mtl and texture, so it's easy to test.

**Comment by BreadLee** [comment]
thanks for your advice! 
Actually,I have already attached  obj, mtl and texture, and I can load it by other project which has lower efficiency.
I noticed that example in your project own only a .bmp or .png texture, and my model owns lots of texture pic.
So Is it possible because this project can't load a 3Dmodel owns more than one pic texture? (I have tried loading some models , all of them have a problem that loading texture. Some can't load completely, some can load only a part of texture )
